src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArr = sourceString.replace(/\n/g, '').split(';').map(element =>
+    element.trim()).filter(element =>
+    element.length > 0);
```

Comment:
Перенось на нові рядкі
```suggestion
  const stylesArr = sourceString.replace(/\n/g, '')
   .split(';')
   .map(element => element.trim())
   .filter(element => element.length > 0);
```